charts/spinnaker/templates/configmap/halyard-config.yaml

```
@@ -109,9 +109,24 @@ data:
     {{- end }}
 
     $HAL_COMMAND config provider kubernetes enable
-    {{- range $name, $account := .Values.kubeConfig.accounts }}
 
-    if $HAL_COMMAND config provider kubernetes account get {{ $name }}; then
+    function custom_resource_exists() {
+      local account="${1}"
+      local kubernetes_kind="${2}"
+      local spinnaker_kind="${3}"
+      echo "${account}" | grep --quiet '"kubernetesKind":"'"${kubernetes_kind}"'","spinnakerKind":"'"${spinnaker_kind}"'"'
```

```suggestion
      echo "${account}" | grep -q '"kubernetesKind":"'"${kubernetes_kind}"'","spinnakerKind":"'"${spinnaker_kind}"'"'
```

The busybox version of `grep` is installed on Halyard, so `--quiet` is not valid. Idem ditto below.